docs/Other SDL Features/Encryption/index.md

```diff
@@ -79,11 +79,14 @@ ServiceEncryptionListener serviceEncryptionListener = new ServiceEncryptionListe
 !@
 
 ## Setting Optional Encryption
-If you want to encrypt a specific RPC, you must configure the payload protected status of the RPC before you send it to the head unit.
+If you want to encrypt a specific RPC, you must configure the payload protected status of the RPC before you send it to the head unit. Before sending RPCs with encryption you should call `startRPCEncryption` on the `sdlManager`.  The best place to put `startRPCEncryption` is in the successful callback of  `startWithReadyHandler`.
 
 @![iOS]
 ##### Objective-C
 ```objc
+[self.sdlManger startRPCEncryption];
```

Comment:
Combine these two code snippets and add a comment to the code snippets. Maybe something like: 
```
// First, start the encryption manager 
code
// Once the encryption manager has started successfully send your encrypted requests
code
```